resources/backend.rb

```diff
@@ -30,6 +30,15 @@
 include SensuCookbook::SensuCommonProperties
 
 property :config, Hash, default: { "state-dir": '/var/lib/sensu/sensu-backend' }
+property :username, String, default: 'admin'
```

Comment:
we need to think through this (handling these credentials properly without exposing them to the process table) and it absolutely won't have a default in this cookbook.

Reply:
Suggest moving them to attributes as `null` and depend on wrapper cookbooks to set them.

Reply:
I've removed the default for username.